docs/deploy-cephfs.md

```diff
@@ -204,8 +204,11 @@ NAME                                 READY     STATUS    RESTARTS   AGE
 pod/csi-cephfsplugin-provisioner-0   4/4       Running   0          25s
 pod/csi-cephfsplugin-rljcv           3/3       Running   0          24s
 
-NAME                                   TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)     AGE
-service/csi-cephfsplugin-provisioner   ClusterIP   10.101.78.75     <none>        8080/TCP   26s
+NAME                                   TYPE        CLUSTER-IP       EXTERNAL-IP
```

Comment:
specially lines like this are not expected to be split

Reply:
Hey, thanks for the review Niels. 
I have changed the ignore_code_blocks parameter to true, so that it will ignore the checks for these cases.

Reply:
Great, that should be fine. No need to add the lin-breaks in those blocks then, so you'll have to undo those changes :wink:

Reply:
Right, updated those.